pkgs/applications/misc/watchmate/default.nix

```diff
@@ -10,22 +10,24 @@
 , wrapGAppsHook4
 , glib
 }:
-
+let 
+    _version = "0.5.2";
```

Comment:
Indentation is off here.  `version'` might be a more idiomatic name.

Reply:
Or `release` or `releaseVersion` if we want to avoid meaningful symbols.

Reply:
sure thing! thanks for the feedback :)